include/alpaka/kernel/TaskKernelGenericSycl.hpp

```diff
@@ -121,7 +122,9 @@ namespace alpaka::experimental
                 cgh};
 
             // register memory fence dummies
-            auto global_fence_dummy = global_fence_buf.get_access(cgh); // Exists once per queue
+            auto global_fence_dummy
+                = global_fence_buf.get_access<sycl::access_mode::read_write, sycl::target::global_buffer>(
```

Comment:
You're right (actually now `global_buffer` has been deprecated and `device` is used instead). We put that because we were getting an error without it, but I've tried now and I don't get the error anymore even with the original version, so we can  keep it as it was.